software/snpsites/main.nf

```diff
@@ -29,7 +29,7 @@ process SNPSITES {
     script:
     def software = getSoftwareName(task.process)
     """
-    snp-sites -c \\
+    snp-sites \\
         $alignment \\
         > filtered_alignment.fas
```

Comment:
```suggestion
    snp-sites \\
        $alignment \\
        $options.args \\
        > filtered_alignment.fas
```

Reply:
We would need `options.args` defined here to provide `-c`?

Reply:
-c could be provided in the `options.args`, yes.
I am not sure I understand the question

Reply:
Need to add `$options.args` to the command as I suggested so we can pass `-c` to the module as an optional argument now instead.

Reply:
Sorry - I thought I'd done that. 🤦 
That was always my intention